### ISSUE TITLE
[demo server] throws a lot of errors (mainly 520, but CF often simply fails with unknown error)

### ISSUE THREAD

**Issue by grinapo** [knadh/listmonk#2040]
The demo.listmonk.app server mainly works, but randomly fails, either doesn't work at all (error on main page) or components fail randomly.

(I do not have the original error since it's being eaten by CF.)

While I understand that it's just a demo it looks weird for my, the firt time curious looker, that the demo server keeps throwing errors. Maybe it could be fixed, because a demo server is usually a good idea.

**Comment by knadh** [comment]
hmm, I'm unable to reproduce this issue despite multiple refreshes on different pages. Are you still able to observe this?

**Comment by grinapo** [comment]
Indeed, right now it's flawless. Maybe the demo server had some problems?
